install.sh

```diff
@@ -96,6 +99,68 @@ process_arguments() {
   fi
 }
 
+pre_flight_checks() {
```

Comment:
Maybe we could add this in E2E test?

Reply:
@jjeongatl we can but only after e2e tests use snapshots like DCAPT does.